data_dir: '/media/tasuku/Samsung_T3/workspace/amr_data/race_track/images' # directory to write image data
save_frequency: 200 # pickle threshold (save data every 'save_frequency')
capacity: 1000000   # number of images to store

# Topics to record and store.
img_topic: '/webcam/image_raw/compressed'
cmd_topic: '/amr_command_2d'
joy_topic: '/joy'

